Rename media page to radio page

pjlee11 opened the pull request:
Resolves #4010

**Overall change:** Changes MediaPage to RadioPage for clarity as we will soon be adding MediaAssetPage which will cause confusion between MediaPage and MediaAssetPage. 

**Code changes:**

- Renames all references to `MediaPage` to `RadioPage`
- Changes the regex variable names from `Media` to `radioAndTv` & `buildMediaRoutes` to `buildRadioAndTvRoutes`
- Changes getInitialData from `media` to `radio` 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- ~[ ] This PR requires manual testing~

Commit:
Replace MediaPage with RadioPage

Commit:
Merge branch 'latest' of github.com:bbc/simorgh into rename-MediaPage-to-RadioPage

Commit:
rename media page to radio page

Commit:
rename media page to radio page

Commit:
Radio Page Main should be in Storybook Main examples

Commit:
Merge branch 'latest' into rename-MediaPage-to-RadioPage

chris-hinds commented on src/app/containers/RadioPage/index.jsx:
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import compose from 'ramda/src/compose';
-import MediaPageMain from '../MediaPageMain';
+import MediaPageMain from '../RadioPageMain';
```
Should this still be `MediaPageMain` ?

chris-hinds reviewed (commented):
Just one comment

Commit:
review changes, RadioContainer and radiopage to radioPage

Commit:
Merge branch 'rename-MediaPage-to-RadioPage' of github.com:bbc/simorgh into rename-MediaPage-to-RadioPage

HarveyPeachey commented on src/app/containers/RadioPage/index.jsx:
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import compose from 'ramda/src/compose';
-import MediaPageMain from '../MediaPageMain';
+import MediaPageMain from '../RadioPageMain';
```
Does `MediaPageMain` need to change to `RadioPageMain`?

HarveyPeachey replied:
Ooh just seen chris's comment

Commit:
ATI, media params should be radio params but type is still media

Commit:
mediaPage -> radioPage

HarveyPeachey reviewed (commented):
LGTM apart from one question

Commit:
filepath change

pjlee11 commented:
@dr3 pointed out that propTypes and ATIAnalytics needed changes as well

Commit:
more renaming

Commit:
Merge branch 'latest' of github.com:bbc/simorgh into rename-MediaPage-to-RadioPage

HarveyPeachey commented on src/app/containers/RadioPageMain/index.test.jsx:
```diff
@@ -2,13 +2,13 @@ import React from 'react';
+import RadioPageMain from '.';
 import amharicPageData from '#data/amharic/bbc_amharic_radio/liveradio';
-import addIdsToBlocks from '../../routes/getInitialData/mediapage/addIdsToBlocks';
+import addIdsToBlocks from '../../routes/getInitialData/radioPage/addIdsToBlocks';
```
Should `radioPage` be camelCase? I'm assuming it shouldn't make a difference

pjlee11 replied:
I think `frontpage` being lower case is a mistake. It's being set as camelCase in the [new preprocessor rules](https://github.com/bbc/simorgh/pull/3998). 

We have camelCase everywhere else so seems like a good time to change it for the radioPage

Commit:
Merge branch 'latest' into rename-MediaPage-to-RadioPage

HarveyPeachey reviewed (approved):
LGTM

Commit:
package.json linting, 2 spaces rather than 4

Commit:
rename ATI params radio -? radioPage

Commit:
so jenkins failed cause git cache didn't see the file rename

Commit:
Merge branch 'rename-MediaPage-to-RadioPage' of github.com:bbc/simorgh into rename-MediaPage-to-RadioPage

Commit:
Merge branch 'latest' of github.com:bbc/simorgh into rename-MediaPage-to-RadioPage

Commit:
properly rename the file

Commit:
rename blah to radioPage

So git local didn't seem to think renaming a directory from `radiopage` to `radioPage` was a commitable change and this was causing Jenkins and Travis to file because the directory never actually updated it's location.

Even after a git cache clean ... GAAAAAH

12 reviewed (approved):
LGTM 👍
